src/include/daos_obj.h

```diff
@@ -900,6 +900,63 @@ daos_obj_anchor_split(daos_handle_t oh, uint32_t *nr, daos_anchor_t *anchors);
 int
 daos_obj_anchor_set(daos_handle_t oh, uint32_t index, daos_anchor_t *anchor);
 
+/**
+ * Open Object Index Table (OIT) of an container
```

Comment:
I think we need to provide more info there that this is a OIT table only for a specific snapshot. To be clear, it returns a special object handle for which we can only call daos_oit_close/list, right? And not the standard object API?

Reply:
Correct, I can add more detailed message in follow-on patches, we have to improve the implementation to support at data protection and sharding, but it wouldn't impact the API.